src/System.Text.Http/System.Text.Http.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Non-allocating HTTP parser</Description>
     <TargetFramework>netstandard1.3</TargetFramework>
     <PackageTags>HTTP parser parsing .NET non-allocating corefxlab</PackageTags>
-    <LangVersion>7.2</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
We already have latest LangVersion in common.props. We should just remove this property.

https://github.com/dotnet/corefxlab/blob/6fb9e0e79789d0f46849a73354ea4d962112ddaa/tools/common.props#L16

Reply:
Fair enough.